src/everything_else/graph.erl

```diff
@@ -7,7 +7,10 @@
 % Edge is directed as follows:  {Parent -> Child} or {From -> To}
```

Comment:
No ale bez przesady z tym everything_else. To jak już to zostawić po prostu bezpośrednio w src

Reply:
ja wohl; nie miałem zielonego pojęcia jak to nazwać, więc wrzuciłem tak. jak nie ma pomysłu na jakąś sensowną nazwę to to wraca do src/